blocks/embed/controller.php

```diff
@@ -30,7 +30,20 @@ public function getBlockTypeDescription()
 
     public function view()
     {
-        $this->set('info', Embed::create($this->source));
+        //$this->set('info', Embed::create($this->source));
+
+        try {
+            $info = Embed::create($this->source);
+
+        } catch (Exception $e) {
+            $info = null;
+            return $e->getMessage();
```

Comment:
You can't pass messages by using `return` keyword in `view()` method of block type controller.
Please use `$this->set()` instead.